central/src/main/java/ai/djl/serving/central/http/BadRequestException.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ * with the License. A copy of the License is located at
+ *
+ * http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+ * OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+package ai.djl.serving.central.http;
+
+/** Thrown when a bad HTTP request is received. */
+public class BadRequestException extends IllegalArgumentException {
```

Comment:
Well it was to say if somehow there is an error where a request did not have all of the parameters that are expected in the request.

Reply:
"serving" uses exactly the same exception for the same purpose I guess. so this would be constistent to "serving", especially when we move central as plugin into "serving" 
+1 from me

Reply:
Yeah, if serving has the same one then we should keep it to prioritize consistency. This refactor could be revisited later once serving and central are merged